server/src/test/java/org/opensearch/cluster/coordination/NoMasterBlockServiceTests.java

```diff
@@ -44,12 +44,12 @@
 
 public class NoMasterBlockServiceTests extends OpenSearchTestCase {
```

Comment:
`NoMasterBlockServiceTests ` -> `NoClusterManagerBlockServiceTests`

Reply:
Changed the method name. 👍